demos/interactive/console.php

```diff
@@ -77,26 +77,25 @@ public function generateReport()
     });
 });
 
-/* THIS DEMO IS DISABLED FOR CLOUD USE
- *
-$t = $tt->addTab('composer update', function ($t) {
-    \atk4\ui\Header::addTo($t, [
-        'icon' => 'terminal',
-        'Command execution',
-        'subHeader' => 'it is easy to run server-side commands and stream output through console',
-    ]);
+if (false) { // THIS DEMO IS DISABLED FOR CLOUD USE
```

Comment:
@romaninsh better to comment code like this way...

Reply:
How this is better?

Reply:
see commits history, legit?

Reply:
Still not sure how it's better because commenting out also goes in commits history, but ... ok, who cares :)

Reply:
It is important you understand it. If you refactor the code (CS, rename, ..., want to find usages, ...) this way it is still standard AST, comment is string, very, very bad!

Reply:
Commits history works either way - but i think this would cause warnings with code reviewers about unreachable code. 

Perhaps we could use environment variable to detect and disable certain things in the cloud, i'll add "$_ENV['CLOUD_DEMO']="1", could you add it instead of "true" then?

Reply:
https://github.com/atk4/infrastructure/blob/master/helm/atk-demo/templates/deployment.yaml#L45

Reply:
> Commits history works either way

Same for you. If you refactor the code (CS, rename, ..., want to find usages, ...) this way it is still standard AST, comment is string, very, very bad! Where refactoring, you do not work with git history at all.

> Perhaps we could use environment variable to detect and disable certain things in the cloud, i'll add "$_ENV['CLOUD_DEMO']="1", could you add it instead of "true" then?

Why not, ok from my side :)

Reply:
@romaninsh done, please check

Also note that "increase coverage by 0.45%". Another reason to not comment code :)

Reply:
changed to much better and secure way using `disable_functions`